docs/misc/design-philosophies.txt

```diff
@@ -254,10 +254,6 @@ enough programming-esque functionality, such as branching and looping, that is
 essential for making presentation-related decisions. The :ref:`Django Template
 Language (DTL) <template-language-intro>` aims to avoid advanced logic.
 
-The Django template system recognizes that templates are most often written by
-*designers*, not *programmers*, and therefore should not assume Python
-knowledge.
```

Comment:
@carltongibson Does it work for you? or you still have [doubts](https://forum.djangoproject.com/t/django-design-philosophies-dtl-confirm-if-should-update/27363/4) and would prefer to rephrase?

Reply:
I think it's OK. The previous paragraph contains the point about aiming _to avoid advanced logic_, so that's probably enough for this document.